Server/image_functions/image_module/contrast_stretching.py

```diff
@@ -0,0 +1,53 @@
+def contrast_stretching(id1, id2):
+    """
+        Process the input image with contrast_stretching
+        and check the image size
+
+        :param id1: the input should be the name string of the image
+        :param id2: the input should be the name string of the processed image
+        :raises ImportError:  if input is not a string
+        :raises NameError:: if the id1 image file does not exist
+        :raises AssertionError: if the image array does not contain
+                three layers
+
+        :returns: a python dictionary contains the base64 string
+                  of the processed image and the image size
+        :rtype: python dictionary
+        """
+    try:
+        import PIL
```

Comment:
is `PIL` in `requirements.txt`?

Reply:
In requirements.txt, the package name is pillow. But when I write functions, I need to write 'import PIL'  instead of ‘import pillow'.
Reference: https://stackoverflow.com/questions/20060096/installing-pil-with-pip

Reply:
also just a suggestion - I don't know why you are getting these errors, but they need to be fixed before we merge